lib/src/app/player.d.ts

```diff
@@ -1,229 +0,0 @@
-declare namespace player {
```

Comment:
Is this no longer needed?

Reply:
I didn't find any place where TypeScript used. Good idea to apply it, but it's a big effort. It's still not done yet, so I suggest to leave this file in git history and get back to it if needed.